canonical-websites/www.ubuntu.com
# Update Ubuntu Advantage Service Description

## Done

Update Ubuntu Advantage Service Description to the latest version

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/ubuntu-advantage/service-description](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description)
- Check against list of highlighted changes in the [copydoc](https://docs.google.com/document/d/1mrNrsC1WDf6QTg3nrVCEZS8vJzGz1fokvoFDVC4s_mI/edit#)


## Issue / Card

Fixes #2282
